testsuite/MDAnalysisTests/core/test_atomselections.py

```diff
@@ -1270,16 +1322,39 @@ def test_similarity_selection_icodes(u_pdb_icodes, selection, n_atoms):
 
     assert len(sel.atoms) == n_atoms
 
-@pytest.mark.parametrize('selection', [
-    'all', 'protein', 'backbone', 'nucleic', 'nucleicbackbone',
-    'name O', 'name N*', 'resname stuff', 'resname ALA', 'type O',
-    'index 0', 'index 1', 'bynum 1-10',
-    'segid SYSTEM', 'resid 163', 'resid 1-10', 'resnum 2',
-    'around 10 resid 1', 'point 0 0 0 10', 'sphzone 10 resid 1',
-    'sphlayer 0 10 index 1', 'cyzone 15 4 -8 index 0',
-    'cylayer 5 10 10 -8 index 1', 'prop abs z <= 100',
-    'byres index 0', 'same resid as index 0',
-])
+
+@pytest.mark.parametrize(
```

Comment:
We generally ask folks to not reformat large bits of code during their PRs as it makes it very hard to review what actually changed. Could you point out which bits you've added to here?

Reply:
It is an reformat came out directly from execution `darker -r upstream/develop testsuite/MDAnalysisTests -L flake8`, after I added a test script `box xz 3 2 4 -5 index 0` into the empty group test. 
Sorry for not aware of the unexpected modification done by darker.